lib/ansiblelint/rules/MissingFilePermissionsRule.py

```diff
@@ -48,5 +48,8 @@ def matchtask(self, file, task):
         if task["action"]["__ansible_module__"] not in self._modules:
             return False
 
+        if task['action'].get('state', None) == "absent":
+            return False
+
         mode = task['action'].get('mode', None)
-        return not isinstance(mode, str)
+        return mode is None
```

Comment:
It's probably cleaner to use
```suggestion
        try:
            task['action']['mode']
        except KeyError:
            return False
        else:
            return True
```
(because with `.get()` it is ambiguous whether it returns a fallback or the key exists and the value actually `is None`)

Reply:
Not in this case, I do still want to to fail if mode is null/empty inside YAML.

Reply:
Another alternative would be to return true only if mode is str or numeric, as that is the desired behavior.